tests/Node/Expression/CallTest.php

```diff
@@ -59,6 +59,10 @@ public function testGetArgumentsWithWrongNamedArgumentNames()
 
     public function testResolveArgumentsWithMissingValueForOptionalArgument()
     {
+        if (version_compare(PHP_VERSION, '8.0', '>=')) {
+            $this->markTestSkipped('substr_compare() has a default value in 8.0, so the test does not work anymore, one should find another PHP built-in function for this test to work in PHP 8.');
```

Comment:
We skip this test as the function signature has been fixed in PHP 8.0. This is an edge case and can only happen for PHP built-in functions that have a discrepancy between its signature and its behavior. Not sure if there is some other function to use in PHP 8.